eng/pipelines/swagger-apiview.yml

```diff
@@ -0,0 +1,37 @@
+parameters:
+  - name: AzureSdkForNetDevOpsFeed
+    type: string
+    default: 'https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json'
+  - name: SwaggerApiParserVersion
+    type: string
+    default: '1.0.0-dev.20240828.2'
+
+trigger: none
+
+jobs:
+- job:
+  pool:
+    name: azsdk-pool-mms-ubuntu-2204-general
+    vmImage: ubuntu-22.04
+
+  steps:
+  - pwsh: |
+      dotnet tool install Azure.Sdk.Tools.SwaggerApiParser `
+        --version ${{ parameters.SwaggerApiParserVersion }}  `
+        --add-source ${{ parameters.AzureSdkForNetDevOpsFeed }} `
+        --global
+    displayName: Install APIView Swagger Parser
+
+  - pwsh: |
+      $(Build.SourcesDirectory)/eng/scripts/Swagger-APIView.ps1 `
+        -TempDirectory $(Agent.TempDirectory) `
+        -ArtiFactsDirectory $(Build.ArtifactStagingDirectory) `
+        -SourceCommitId $(System.PullRequest.SourceBranch) `
```

Comment:
Do these need to be source and target? Can they not be the default of HEAD and HEAD^ which is what we use in our helper function defaults?